spec/controllers/activities_controller_spec.rb

```diff
@@ -80,7 +80,7 @@
       end
 
       describe 'empty filter selection' do
-        before { get 'index', apply: true }
+        before do get 'index', apply: true end
```

Comment:
Prefer {...} over do...end for single-line blocks.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.